ui/js/models/perf/alerts.js

```diff
@@ -360,5 +361,31 @@ treeherder.factory('PhAlerts', [
                     });
                 });
             },
+            findPushIdNeighbours: (dataPoint, resultSetData, direction) => {
+                const pushId = dataPoint.resultSetId;
+                let pushIdIndexOf;
+                let relativePos;
+
+                // retriggers duplicate pushes
+                if (direction === 'left') {
+                    pushIdIndexOf = 'indexOf';
+                    relativePos = -1;
+                } else {
+                    pushIdIndexOf = 'lastIndexOf';
+                    relativePos = 1;
+                }
+
+                const pushIdIndex = resultSetData[pushIdIndexOf](pushId);
+                return {
+                    push_id: resultSetData[pushIdIndex + relativePos],
+                    prev_push_id: resultSetData[pushIdIndex + (relativePos - 1)],
+                };
+            },
+            nudgeAlert: (dataPoint, towardsDataPoint) => {
+                console.log('nudgeAlert(dataPoint, towardsDataPoint):'); // eslint-disable-line no-console
+                console.log(JSON.stringify(dataPoint)); // eslint-disable-line no-console
```

Comment:
> if you just want feedback for a work in progress it is ok if travis fails

I'm guessing the eslint-disable was because this PR was tested on prototype, and so otherwise `yarn build` would have failed, preventing the deploy. 

However I agree adding the eslint-disable is asking for the `console.log()`s to end up being accidentally merged to master, so using the the debugger is probably a better approach:
https://hackernoon.com/please-stop-using-console-log-its-broken-b5d7d396cf15

Reply:
That article feels like opinion masquerading as authority. I've been using console debugging for years (along with other techniques) and have no intention of stopping. :)

Can we remove eslint from the `yarn build` step? We should lint before merging into master, but I don't see why passing lint should be a prerequisite for deploying.

Reply:
It wasn't the article I've read before, but the first that came up on google just now - so might not be the best. However I really do think people need to learn the debugger as **one** of the tools in their toolbox - if they know it and then console debugging is still the best tool for a particular situation, then that's fine - but I suspect in some instances it's actually a case of sticking with something they are more familiar with, which is less fine.

> Can we remove eslint from the yarn build step? We should lint before merging into master, but I don't see why passing lint should be a prerequisite for deploying in a test environment.

I don't think we want to do that. People can accidentally merge to `master` even when Travis is failing - eg interaction with another change on master, post rebase/merge. Also, we're not running eslint per se, but it's loaded via eslint-loader during the webpack build (still possible to disable, but it's not a case of us having the build entry in package.json be `"build": "yarn lint && neutrino build"`).